doc/nrf/security.rst

```diff
@@ -1,17 +1,56 @@
+.. _security:
 .. _security_index:
 
 Security
 ########
 
-The |NCS| provides a variety of security features.
-Make sure to check the :ref:`overview of available options <security>` and consider enabling :ref:`access port protection mechanism <app_approtect>`.
+This section provides an overview of core security features available in Nordic Semiconductor products.
+The features are made available either as built-ins in modules, drivers, and subsystems, or are shown in samples or applications in |NCS|.
 
-If you are working with nRF53 Series or nRF91 Series devices, use :ref:`Trusted Firmware-M <ug_tfm>` to increase the security of your end product.
+The following table lists the available general security features.
+Some of them are documented in detail in other parts of this documentation, while others are documented in the subpages in this section.
+
+ .. list-table::
+  :header-rows: 1
+
+  * - Security feature
+    - Description
+    - Configuration
+    - Available dedicated components
```

Comment:
Maybe related instead of dedicated? Available related components.
Or resources... related resources.
I will leave it up to you if you want to do any adjustments here.
As I cannot quite put my finger on what exactly I was reacting on when I read Available dedicated components, but it just triggered something LOL

Reply:
These are software components, as on the figure in https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev.html . I went for `Related components`.